src/components/Categories.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react'
+import Link from './Link'
+import './../css/categories.css'
+
+// product type def - need to make global
+interface Product {
+  id: string
```

Comment:
This still needs to move to a global type. If you want you can resolve PR conversations as you complete them in the code to keep track of what's done or not. I commonly need to do that on my PRs haha.